README.md

```diff
@@ -204,12 +204,18 @@ A breakdown of the options and methods available to the SDK.
   ```javascript
   steps: [
     {
-      type:'welcome',
-      options:{
-        title:'Open your new bank account'
+      type: 'welcome',
+      options: {
+        title: 'Open your new bank account'
+      }
+    },
+    {
+      type: 'document',
+      options: {
+        useWebcam: true // This is in beta!
```

Comment:
I disagree, it's nested as part of the document step and so logically refers only to that step.

I think it's simpler to have clear names across the steps. e.g. we use `options` rather than `docOptions`